docs/azure_arc_jumpstart/azure_arc_app_svc/aks/aks_azure_apimgmt_arm_template/_index.md

```diff
@@ -0,0 +1,181 @@
+
+    ![New Azure resource group with all resources](./02.png)
+
+## Windows Login & Post Deployment
+
+* Now that first phase of the automation is completed, it is time to RDP to the client VM using it's public IP.
+
+    ![Client VM public IP](./03.png)
+
+* At first login, as mentioned in the "Automation Flow" section above, the [_AppServicesLogonScript_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_app_services_jumpstart/aks/arm_template/artifacts/AppServicesLogonScript.ps1) PowerShell logon script will start it's run.
+
+* Let the script to run its course and **do not close** the PowerShell session, this will be done for you once completed. Once the script will finish it's run, the logon script PowerShell session will be closed, the Windows wallpaper will change and the Azure web application will be deployed on the cluster and be ready to use.
+
+    > **Note: As you will notices from the screenshots below, during the Azure Arc enabled app services environment, the _log-processor_ service pods will be restarted and will go trough multiple Kubernetes pod lifecycle stages. This is normal and can safely be ignored. To learn more about the various Azure Arc enabled app services Kubernetes components, visit the official [Azure Docs page](https://docs.microsoft.com/en-us/azure/app-service/overview-arc-integration#pods-created-by-the-app-service-extension).**
+
+    ![PowerShell logon script run](./04.png)
+
+    ![PowerShell logon script run](./05.png)
+
+    ![PowerShell logon script run](./06.png)
+
+    ![PowerShell logon script run](./07.png)
+
+    ![PowerShell logon script run](./08.png)
+
+    ![PowerShell logon script run](./09.png)
+
+  Once the script finishes it's run, the logon script PowerShell session will be closed, the Windows wallpaper will change, and both the API Management gateway and the sample API will be configured on the cluster.
+
+    ![Wallpaper change](./10.png)
+
+* Since this scenario is deploying both the Azure API Management instance, the API Management gateway extension and configure a sample API, you will also notice additional, newly deployed Azure resources in the resources group (at this point you should have **12 various Azure resources deployed**. The important ones to notice are:
+
+  * **Azure Arc enabled Kubernetes cluster** - Azure Arc enabled app services are using this resource to deploy the app services [cluster extension](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/conceptual-extensions), as well as using Azure Arc [Custom locations](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/conceptual-custom-locations).
+
+  * [**Azure API Management service**](https://docs.microsoft.com/en-us/azure/api-management/) - An instance of Azure API Management needs to be created with a gateway resource befor you can enable the API Management gateway extension.
+
+  ![Additional Azure resources in the resource group](./11.png)
+
```

Comment:
I think we should have a section here showing what is it you can do and show sample API calls and open some APIM screens.